homework-submission/methods.js

```diff
@@ -0,0 +1,10 @@
+function readFile(path) {
+  return new Promise((resolve, reject) => {
```

Comment:
You don't need to return a new Promise in order to get the promise!

`fetch` contains the promise you have asked for, I would consider returning it instead!

Reply:
Relevant changes are made!